proto/cosmwasm/wasm/v1/proposal.proto

```diff
@@ -135,18 +135,6 @@ message UnpinCodesProposal {
   repeated uint64 code_ids = 3 [(gogoproto.customname) = "CodeIDs", (gogoproto.moretags) = "yaml:\"code_ids\""];
 }
 
-// UpdateStatusProposal gov proposal content type to update the contract status.
-message UpdateContractStatusProposal {
```

Comment:
Don't we need any more?

Reply:
Yes, this proposal's role is to modify the smart contract as inactive or active. This is the one of contract blacklist functions. I'll add the contract blacklist function through another PR.

Reply:
Does it mean that the blacklist functions will replace the msg we previously added on our own?

Reply:
@shiki-tak , Yes, I'll have a plan to add the contract blacklist data in the genesisState. So I'll add it to sixth index in the genesisState like below. 
```proto
message GenesisState {
   ….

  // GenMsgs has wasmd sdk type messages to execute in the genesis phase
  repeated cosmwasm.wasm.v1.GenesisState.GenMsgs  gen_msgs  = 5 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "gen_msgs,omitempty"];

  // InactiveContractAddresses is a list of contract address that set inactive
  repeated string inactive_contract_addresses = 6 [(gogoproto.jsontag) = "inactive_contract_address, omitempty"];
}
```